test/unit/middleware/check_platform_middleware_test.dart

```diff
@@ -1,31 +1,27 @@
 import 'package:crowdleague/actions/device/check_platform.dart';
```

Comment:
this is refactoring this test to use our new approach to testing middleware, rather then checking that the state is updated, we are checking that our mock store has dispatched what we want it to!

Reply:
Thanks for the clean up - looks great!